settings/default/map.lua

```diff
@@ -105,14 +105,14 @@ xi.settings.map =
     -- Enable/disable jobs other than BST and RNG having widescan
     ALL_JOBS_WIDESCAN = true,
 
-    -- Modifier to apply to player speed. 0 is the retail accurate default. Negative numbers will reduce it.
+    -- Modifier to apply to player speed. 0 is the retail accurate default. Negative numbers will reduce it. This affects all entity base speed as well as max speed.
     SPEED_MOD = 0,
 
     -- Modifier to apply to mount speed. 0 is the retail accurate default. Negative numbers will reduce it.
     -- Note retail treats the mounted speed as double what it actually is.
     MOUNT_SPEED_MOD = 0,
 
-    -- Modifier to apply to agro'd monster speed. 0 is the retail accurate default. Negative numbers will reduce it.
+    -- Modifier to apply to agro'd monster speed (i.e. while engaged in combat). 0 is the retail accurate default. This is a percentage.
```

Comment:
if its a percent we should put a float value here and just give it a range starting at 1.00, making 0.5 half speed and 2.0 double speed. I just say that because that is consistent with other % adjusters in the settings and help visually differentiate it form the 2 speed settings that do not work that way.

Reply:
for github readers: mow and I keep being naughty and discussing things in discord and not github. revolving this current discussion.